workflows/utilities/data_import/wf_update_srr_metadata.wdl

```diff
@@ -0,0 +1,23 @@
+version 1.0
+
+import "../../../tasks/utilities/data_handling/task_fetch_srr_metadata.wdl" as srr_task
+
+workflow wf_retrieve_srr {
+  meta {
```

Comment:
meta field is something that is not part of our style-guide. This warrants a discussion if it's something that we want to start adding,

Reply:
![image](https://github.com/user-attachments/assets/84e7e88d-a699-4bf9-ade2-2a01e9f54439)

Reply:
Its in our style guide to include meta field